portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,103 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Portfolio: Tanvi Ohri</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<h1>Tanvi Ohri</h1>
+		<img src="/images/Tanvi.jpg" max-width="700" height="350">
+		<p>Click here for a random greeting:</p>
+		<button onclick="addRandomGreeting()">Hello</button>
+		<div id="greeting-container"></div>
+		<p>Welcome to my Portfolio! I am a third->fourth year student from IIT Guwahati and an alumnus of Delhi Public
+			School, Dwarka.</p>
+		<h2>Internships and Projects</h2>
+		<h4> ARES </h4>
+		<h5>Department of Artificial Intelligence, University of Malaya </h5>
+		<img src="/images/Ares.jpg" max-width="700" height="350">
+		<ul>
+			<li>Did a comparative analysis of various 2D and 3D mapping techniques using various sensor combinations,
+				SLAM libraries, and fine-tuned parameters.</li>
+			<li>Implemented autonomous navigation and Frontier Based Exploration Technique (for autonomous mapping) in a
+				2D map.</li>
+		</ul>
+		<h4> ns-3 Network Simulator </h4>
+		<h5>Networks Lab, IIT Guwahati </h5>
+		<ul>
+			<li>Did a comparative analysis of the performance of TCP congestion avoidance algorithms over wired and
+				wireless networks using different applications.</li>
+		</ul>
+		<h4> Socket Programming </h4>
+		<h5>Networks Lab, IIT Guwahati </h5>
+		<ul>
+			<li>Used Client-Server socket programming to implement a basic banking system which implements functionality
+				to handle user authentication and credit, debit or statement requests.</li>
+		</ul>
+		<h4> Automated Robot For Library Enhancement </h4>
+		<h5>4I Labs, IIT Guwahati </h5>
+		<img src="/images/Arle.jpg" max-width="700" height="350">
+		<ul>
+			<li>Implemented mapping, localization and autonomous navigation in a 2D environment using LIDAR and visual
+				odometry.</li>
+		</ul>
+		<h4> Mutual Fund Analysis </h4>
+		<h5>Dimension, Dept. of Mathematics, IIT Guwahati </h5>
+		<ul>
+			<li>Analysed the data of over 100 mutual fund schemes quantitatively and qualitatively.</li>
```

Comment:
Interesting, did you pursue it further and started investing in stock markets? May be you can start by opening an account on https://zerodha.com/

Reply:
Haha! I didn't then, probably now I can try with the stipend :)